CCXDEV-7143 Assessing and managing issues with advisor (#41968)
openshift/openshift-docs: modules/enabling-insights-advisor-recommendations.adoc

// Module included in the following assemblies:
//
// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc

:_content-type: PROCEDURE
[id="enabling-insights-advisor-recommendations_{context}"]
= Enabling a previously disabled Insights Advisor recommendation

> When a recommendation is disabled for all clusters, you will no longer see the recommendation in Insights Advisor. To reverse this action, complete the following steps:

Reviewer: ```suggestion
When a recommendation is disabled for all clusters, you no longer see the recommendation in Insights Advisor. You can change this behavior.
```

.Prerequisites

* Remote health reporting is enabled, which is the default.
* Your cluster is link:https://console.redhat.com/openshift/register[registered] on Red Hat Hybrid Cloud Console.
* You are logged in to link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].

.Procedure

. Navigate to *Advisor* -> *Recommendations* on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].

> . Filter the recommendations by *Status* -> *Disabled*

Reviewer: Missing a period.

. Locate the recommendation to enable.

> . Click the *Options* menu {kebab}, and click *Enable recommendation*.

Reviewer: ```suggestion
. Click the *Options* menu {kebab}, and then click *Enable recommendation*.
```
